translate/rus.json

```diff
@@ -106,7 +106,7 @@
 "dark":"темный",
 "date":"дата",
 "DE":"СЛВ",
-"decisive openings":"результативные дебюты",
+"decisive openings":"результативные дебюты", 
```

Comment:
I think this line can be removed entirely since "Win & Draw" stat covers everything that this line does

Reply:
Win & Draw = 1.5 score, but not 2.0
Decisive = win & draw + double wins.

Reply:
Ok, I see, good thing I reverted it:)